test/pages/api/xAPI/statements/index.spec.ts

```diff
@@ -1,396 +1,198 @@
-import sinon from 'sinon';
```

Comment:
I'd love to go over these changes with you so I can understand your approach test development, it looks much cleaner!